{"name": "test_valid_credentials_2[tomsmith-SuperSecretPassword!]", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <regular.tests.test_selenium_elemental.TestLogin object at 0x107f30750>\ndriver = <selenium.webdriver.remote.webdriver.WebDriver (session=\"0b73a1154aab61c8459e274c950ba23a\")>\n\n    @pytest.fixture\n    def login(self, driver):\n>       return login_page.LoginPage(driver)\n\ntests/test_selenium_elemental.py:8: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <regular.pages.login_page.LoginPage object at 0x107f17110>\ndriver = <selenium.webdriver.remote.webdriver.WebDriver (session=\"0b73a1154aab61c8459e274c950ba23a\")>\n\n    def __init__(self, driver):\n        self.driver = driver\n        self._visit(\"/login\")\n>       assert self._is_displayed(self._login_form)\nE       AssertionError\n\npages/login_page.py:16: AssertionError"}, "parameters": [{"name": "name", "value": "'tomsmith'"}, {"name": "passw", "value": "'SuperSecretPassword!'"}], "start": 1614289425847, "stop": 1614289425847, "uuid": "c058fd1c-72b3-4603-b45f-1574cb36001c", "historyId": "ed3587293625da7159a618d8bf30ef6d", "testCaseId": "a375d38c0b44bedf24d3487062073299", "fullName": "tests.test_selenium_elemental.TestLogin#test_valid_credentials_2", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_selenium_elemental"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "C07ZQ0G3JYVX"}, {"name": "thread", "value": "90508-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_selenium_elemental"}]}